tracee-ebpf/tracee/argprinters.go

```diff
@@ -115,6 +115,14 @@ func (t *Tracee) parseArgs(ctx *context, args map[string]interface{}) error {
 				args["type"] = typeIdStr
 			}
 		}
+	case SchedProcessExecEventID:
+		if stdin, isUint32 := args["stdin"].(uint32); isUint32 {
```

Comment:
If for any reason it is not uint32, don't we want to maybe notify or log something (maybe at debug level). It might help us in future if something changed between kernel versions, or between distributions, not critical - just an idea. What do you think?

Reply:
actually this shouldn't change as these values are something we define. (see #define FILE_TYPE_SOCK in tracee.bpf.c).